sdk/storage/azure-storage-queue/azure/storage/queue/_message_encoding.py

```diff
@@ -133,33 +134,12 @@ def decode(self, content, response):
                 error=error)
 
 
-class TextXMLEncodePolicy(MessageEncodePolicy):
-    """XML message encoding policy for text messages.
-
-    Encodes text (unicode) messages to XML. If the input content
-    is not text, a TypeError will be raised.
-    """
-
-    def encode(self, content):
-        if not isinstance(content, six.text_type):
-            raise TypeError("Message content must be text for XML encoding.")
-        return xml_escape(content)
-
-
-class TextXMLDecodePolicy(MessageDecodePolicy):
-    """Message decoding policy for XML-encoded messages into text.
-
-    Decodes XML-encoded messages to text (unicode).
-    """
-
-    def decode(self, content, response):
-        return xml_unescape(content)
-
-
 class NoEncodePolicy(MessageEncodePolicy):
     """Bypass any message content encoding."""
 
     def encode(self, content):
+        if isinstance(content, six.binary_type) and sys.version_info > (3,):
+            raise ValueError("Message content must not be bytes.")
```

Comment:
Raise TypeError, that was the previous behavior anyway (see the update in tests you did)